lerna.json

```diff
@@ -7,7 +7,6 @@
   "useWorkspaces": true,
   "command": {
     "version": {
-      "allowBranch": "main",
       "conventionalCommits": true,
       "createRelease": "github",
```

Comment:
can we override this when doing a prerelease? this config creates release objects in github which i think is unnecessary

Reply:
What kind of release objects?

Reply:
https://github.com/amplitude/Amplitude-TypeScript/releases

Reply:
Not sure why deleting `allowBranch` here will create release objects because [`--allow-branch`](https://github.com/lerna/lerna/tree/main/libs/commands/version#--allow-branch-glob) is used to enable `lerna version` on the specified feature branch. 

It looks like that [`--create-release`](https://github.com/lerna/lerna/tree/main/libs/commands/version#--create-release-type) is the option to create Github release. To make sure no release objects are made, add `--no-changelog` here to disable the generation (or updating) of CHANGELOG.md files.

Anyway, revert the change in `lerna.json` and pass `--allow-branch` when prerelease.

Reply:
This comment is for line 11:

```
"createRelease": "github",
```

Reply:
to clarify, my comments above aren't for the line

```
"allowBranch": "main",
```

i left them for line 11, but can see how github renders comments may be confusing 🤷 

we need that to allow releases from feature branches.

Reply:
Resolve this conversation as it's continued in the next conversation.